addons/im_livechat/static/src/core/web/discuss_sidebar_categories_livechat.js

```diff
@@ -5,7 +5,10 @@ import { discussSidebarCategoriesRegistry } from "@mail/discuss/core/web/discuss
 discussSidebarCategoriesRegistry.add(
     "livechats",
     {
-        predicate: (store) => store.discuss.livechat.threads.some((thread) => thread?.is_pinned),
+        predicate: (store) =>
+            store.discuss.livechat.threads.some(
+                (thread) => thread?.displayToSelf || thread?.eq(store.discuss.thread)
```

Comment:
In 17.0 this condition is redundant, shall we still keep it in prevision of the "interactable" threads?